components/carousel/script.js

```diff
@@ -166,6 +190,27 @@ class Carousel extends BaseComponent {
         this.createPaginationControls();
     }
 
+    /**
+     * Returns the current page
+     */
+    get currentPage() {
+        return this.currentItemIndex / this.pageSize + 1;

```

Comment:
Should we add precision or cap the numbers after the whole number, what do you think?

Reply:
Yes - changed.